tests/main/interfaces-hooks/task.yaml

```diff
@@ -10,8 +11,32 @@ prepare: |
 restore: |
     rm -f basic-iface-hooks-consumer_1.0_all.snap
     rm -f basic-iface-hooks-producer_1.0_all.snap
+    rm -f "$CONSUMER_DATA/prepare-plug-consumer-done"
+    rm -f "$PRODUCER_DATA/prepare-slot-producer-done"
+    rm -f "$CONSUMER_DATA/connect-plug-consumer-done"
+    rm -f "$PRODUCER_DATA/connect-slot-producer-done"
+
+environment:
```

Comment:
Nitpick, move this immediately after summary so that human readers will first read the environment section and then see the referenced variables in the restore section.